project/driver_controls/src/debounce.c

```diff
@@ -0,0 +1,34 @@
+#include "debounce.h"
```

Comment:
This debounce isn't great. Can we just remove it for now and add debouncing later?